perform-release.sh

```diff
@@ -28,20 +31,23 @@ for f in $(find . -name 'pom.xml' -not -path '*target*'); do
 done
 mvn versions:set -DallowSnapshots=true -DgenerateBackupPoms=false -DnewVersion=$RELEASE_VERSION
 
-source change-scala-versions.sh 2.10
-source change-cuda-versions.sh 7.5
-mvn clean deploy -Dgpg.executable=gpg2 -DperformRelease -Psonatype-oss-release -DskipTests -DstagingRepositoryId=$STAGING_REPOSITORY
-source change-scala-versions.sh 2.11
-source change-cuda-versions.sh 8.0
-mvn clean deploy -Dgpg.executable=gpg2 -DperformRelease -Psonatype-oss-release -DskipTests -DstagingRepositoryId=$STAGING_REPOSITORY
-source change-spark-versions.sh 2
-mvn clean deploy -Dgpg.executable=gpg2 -DperformRelease -Psonatype-oss-release -DskipTests -DstagingRepositoryId=$STAGING_REPOSITORY -Dspark.major.version=2
-
-source change-spark-versions.sh 1
-source change-scala-versions.sh 2.11
-source change-cuda-versions.sh 8.0
+if [[ "${SKIP_BUILD}" == "0" ]]; then
+    source change-scala-versions.sh 2.10
```

Comment:
There are no modules that use both CUDA and Spark, so that's not an issue for that.

I've replied for all the rest at https://github.com/deeplearning4j/Arbiter/pull/93#pullrequestreview-54745066